bug

ryan-ally opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x ] Tests for the changes have been added (for bug fixes / features)
- [ x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

snyk-bot committed:
fix: upgrade typescript from 4.0.2 to 4.1.3

Snyk has created this PR to upgrade typescript from 4.0.2 to 4.1.3.

See this package in npm:
https://www.npmjs.com/package/typescript

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/3d020cfc-0fe7-477b-9564-03c6c54e546d?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: package.json & yarn.lock to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-LODASH-590103

snyk-bot committed:
fix: package.json & yarn.lock to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-WEBSOCKETEXTENSIONS-570623

snyk-bot committed:
fix: package.json & yarn.lock to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-571484

snyk-bot committed:
fix: upgrade tslib from 2.0.1 to 2.0.3

Snyk has created this PR to upgrade tslib from 2.0.1 to 2.0.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/25abe1dd-8e18-42b6-846e-f3af388ed2a5?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade systemjs from 0.18.10 to 0.21.6

Snyk has created this PR to upgrade systemjs from 0.18.10 to 0.21.6.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade @microsoft/api-extractor from 7.7.11 to 7.12.1

Snyk has created this PR to upgrade @microsoft/api-extractor from 7.7.11 to 7.12.1.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade @types/node from 12.12.34 to 12.19.9

Snyk has created this PR to upgrade @types/node from 12.12.34 to 12.19.9.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: aio/aio-builds-setup/dockerbuild/scripts-js/package.json, aio/aio-builds-setup/dockerbuild/scripts-js/yarn.lock & aio/aio-builds-setup/dockerbuild/scripts-js/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/npm:debug:20170905

ryan-ally committed:
Merge pull request #9 from ryan-ally/snyk-fix-f9373baa55a55b348d1af794fcd8abfc

[Snyk] Fix for 1 vulnerabilities

HitomiTenshi commented:
I just randomly looked through PRs and saw this. Is this supposed to be some kind of ad for Synk or what are you trying to achieve with this? What bug does this PR solve? The sheer lack of information in this PR baffles me.

snyk-bot committed:
fix: package.json, yarn.lock & .snyk to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-JS-AJV-584908
- https://snyk.io/vuln/SNYK-JS-ELLIPTIC-571484
- https://snyk.io/vuln/SNYK-JS-HTTPPROXY-569139
- https://snyk.io/vuln/SNYK-JS-JQUERY-174006
- https://snyk.io/vuln/SNYK-JS-JQUERY-565129
- https://snyk.io/vuln/SNYK-JS-JQUERY-567880
- https://snyk.io/vuln/SNYK-JS-LODASH-567746
- https://snyk.io/vuln/SNYK-JS-LODASH-590103
- https://snyk.io/vuln/SNYK-JS-LODASH-608086
- https://snyk.io/vuln/SNYK-JS-MINIMIST-559764
- https://snyk.io/vuln/SNYK-JS-NODEFETCH-674311
- https://snyk.io/vuln/SNYK-JS-NODEFORGE-598677
- https://snyk.io/vuln/SNYK-JS-SERIALIZEJAVASCRIPT-570062
- https://snyk.io/vuln/SNYK-JS-WEBSOCKETEXTENSIONS-570623
- https://snyk.io/vuln/SNYK-JS-Y18N-1021887
- https://snyk.io/vuln/npm:jasmine-core:20180216
- https://snyk.io/vuln/npm:qs:20170213


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

ryan-ally committed:
Merge pull request #10 from ryan-ally/snyk-fix-faed02702ff71cae1b395eb5270c590d

[Snyk] Fix for 17 vulnerabilities

snyk-bot committed:
fix: package.json, yarn.lock & .snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

ryan-ally committed:
Merge pull request #11 from ryan-ally/snyk-fix-c1f125627e70f4862722ec9561e6ec27

[Snyk] Fix for 1 vulnerabilities

ryan-ally committed:
Merge branch 'master' into snyk-fix-b1fb6c4e9886a1df10c146007be1a0ea

ryan-ally committed:
Merge pull request #4 from ryan-ally/snyk-fix-b1fb6c4e9886a1df10c146007be1a0ea

[Snyk] Security upgrade @angular-devkit/build-angular from 0.1100.0-rc.1 to 0.1100.0

ryan-ally committed:
Merge branch 'master' into snyk-fix-b684b910fb7734d6ce4061546a485f7d

ryan-ally committed:
Merge pull request #3 from ryan-ally/snyk-fix-b684b910fb7734d6ce4061546a485f7d

[Snyk] Security upgrade @angular-devkit/build-angular from 0.1100.0-rc.1 to 0.1100.0

ryan-ally committed:
Merge branch 'master' into snyk-fix-d57cd52cada5df7ca704d0e400632a49

ryan-ally committed:
Merge pull request #2 from ryan-ally/snyk-fix-d57cd52cada5df7ca704d0e400632a49

[Snyk] Security upgrade @angular-devkit/build-angular from 0.1100.0-rc.1 to 0.1100.0

ryan-ally committed:
Merge pull request #8 from ryan-ally/snyk-upgrade-a9b808428c7c43c6bf9feda9838aa58c

[Snyk] Upgrade @types/node from 12.12.34 to 12.19.9

ryan-ally committed:
Merge branch 'master' into snyk-upgrade-b7d0fc04a093c8a30067b60d06364aee

ryan-ally committed:
Merge pull request #7 from ryan-ally/snyk-upgrade-b7d0fc04a093c8a30067b60d06364aee

[Snyk] Upgrade @microsoft/api-extractor from 7.7.11 to 7.12.1

ryan-ally committed:
Merge pull request #6 from ryan-ally/snyk-upgrade-760f8ea55fb621fda25a470101ebccda

[Snyk] Upgrade systemjs from 0.18.10 to 0.21.6

ryan-ally committed:
Merge branch 'master' into snyk-upgrade-73fe3a82a60dd1369b223b1fe2bbd896

ryan-ally committed:
Merge pull request #5 from ryan-ally/snyk-upgrade-73fe3a82a60dd1369b223b1fe2bbd896

[Snyk] Upgrade tslib from 2.0.1 to 2.0.3

ryan-ally committed:
Merge pull request #1 from ryan-ally/snyk-upgrade-fff5b2972638b17ec8c33b976202ff78

[Snyk] Upgrade typescript from 4.0.2 to 4.1.3

snyk-bot committed:
fix: aio/tools/examples/shared/package.json, aio/tools/examples/shared/yarn.lock & aio/tools/examples/shared/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

snyk-bot committed:
fix: aio/tools/examples/shared/package.json, aio/tools/examples/shared/yarn.lock & aio/tools/examples/shared/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

snyk-bot committed:
fix: upgrade rollup from 2.16.1 to 2.34.2

Snyk has created this PR to upgrade rollup from 2.16.1 to 2.34.2.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade @octokit/rest from 16.28.7 to 16.43.2

Snyk has created this PR to upgrade @octokit/rest from 16.28.7 to 16.43.2.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - @babel/cli from 7.8.4 to 7.12.10.
    See this package in npm: 
  - @babel/core from 7.9.0 to 7.12.10.
    See this package in npm: 
  - @babel/generator from 7.9.4 to 7.12.10.
    See this package in npm: 
  - @babel/preset-env from 7.10.2 to 7.12.10.
    See this package in npm: 
  - @babel/template from 7.8.6 to 7.12.7.
    See this package in npm: 
  - @babel/traverse from 7.9.0 to 7.12.10.
    See this package in npm: 
  - @babel/types from 7.9.0 to 7.12.10.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade source-map-support from 0.5.9 to 0.5.19

Snyk has created this PR to upgrade source-map-support from 0.5.9 to 0.5.19.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade @types/selenium-webdriver from 3.0.7 to 3.0.17

Snyk has created this PR to upgrade @types/selenium-webdriver from 3.0.7 to 3.0.17.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/afdb863b-71e4-44cc-a8c4-c2b751d20611?utm_source=github&utm_medium=upgrade-pr

ryan-ally committed:
Merge pull request #13 from ryan-ally/snyk-fix-da73ab7de129a837cffc837ccb9980d7

[Snyk] Fix for 1 vulnerabilities

ryan-ally committed:
Merge pull request #18 from ryan-ally/snyk-upgrade-8fba790d937670b8376f47848a4613f1

[Snyk] Upgrade @types/selenium-webdriver from 3.0.7 to 3.0.17

ryan-ally committed:
Merge branch 'master' into snyk-fix-07fa8bdcfb3f9cdfc5fc2a065fc87211

snyk-bot committed:
fix: aio/tools/examples/shared/package.json, aio/tools/examples/shared/yarn.lock & aio/tools/examples/shared/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

ryan-ally committed:
Merge pull request #12 from ryan-ally/snyk-fix-07fa8bdcfb3f9cdfc5fc2a065fc87211

[Snyk] Fix for 1 vulnerabilities

ryan-ally committed:
Merge branch 'master' into snyk-fix-3b6546fbb55f47c1b1e27a44b569bde4

ryan-ally committed:
Merge pull request #19 from ryan-ally/snyk-fix-3b6546fbb55f47c1b1e27a44b569bde4

[Snyk] Fix for 1 vulnerabilities

snyk-bot committed:
fix: aio/tools/examples/shared/package.json, aio/tools/examples/shared/yarn.lock & aio/tools/examples/shared/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

ryan-ally committed:
Merge pull request #17 from ryan-ally/snyk-upgrade-148362e931708312c97e16066c3475e3

[Snyk] Upgrade source-map-support from 0.5.9 to 0.5.19

ryan-ally committed:
Merge pull request #16 from ryan-ally/snyk-upgrade-bee85da87271e388d1b34cb23fb54e0d

[Snyk] Upgrade: @babel/cli, @babel/core, @babel/generator, @babel/preset-env, @babel/template, @babel/traverse, @babel/types

ryan-ally committed:
Merge pull request #15 from ryan-ally/snyk-upgrade-230a9d54b612a7348fa80da90f6051c5

[Snyk] Upgrade @octokit/rest from 16.28.7 to 16.43.2

ryan-ally committed:
Merge pull request #14 from ryan-ally/snyk-upgrade-6388faae5452f421fa4324b80e69fbb2

[Snyk] Upgrade rollup from 2.16.1 to 2.34.2

ryan-ally committed:
Merge pull request #20 from ryan-ally/snyk-fix-8d10d79cba328d11c955b4460d03b473

[Snyk] Fix for 1 vulnerabilities

snyk-bot committed:
fix: aio/tools/examples/shared/package.json, aio/tools/examples/shared/yarn.lock & aio/tools/examples/shared/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

ryan-ally committed:
Merge pull request #21 from ryan-ally/snyk-fix-0f873d0bb796047e2c92f1a5593c65c5

[Snyk] Fix for 1 vulnerabilities

snyk-bot committed:
fix: package.json, yarn.lock & .snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

snyk-bot committed:
fix: aio/tools/examples/shared/package.json, aio/tools/examples/shared/yarn.lock & aio/tools/examples/shared/.snyk to reduce vulnerabilities


The following vulnerabilities are fixed with a Snyk patch:
- https://snyk.io/vuln/SNYK-JS-LODASH-567746

snyk-bot committed:
fix: upgrade zone.js from 0.10.3 to 0.11.3

Snyk has created this PR to upgrade zone.js from 0.10.3 to 0.11.3.

See this package in npm:
https://www.npmjs.com/package/zone.js

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/947978ad-37f3-49eb-904c-2c373fa017c8?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - angular from 1.8.0 to 1.8.2.
    See this package in npm: https://www.npmjs.com/package/angular
  - angular-route from 1.8.0 to 1.8.2.
    See this package in npm: https://www.npmjs.com/package/angular-route

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/947978ad-37f3-49eb-904c-2c373fa017c8?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade source-map from 0.6.1 to 0.7.3

Snyk has created this PR to upgrade source-map from 0.6.1 to 0.7.3.

See this package in npm:
https://www.npmjs.com/package/source-map

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/1e234a89-5661-4095-a027-dcbc8e328cfe?utm_source=github&utm_medium=upgrade-pr

ryan-ally committed:
Merge pull request #23 from ryan-ally/snyk-fix-5cc361f870b07ac8c1f64326c977c678

[Snyk] Fix for 1 vulnerabilities

ryan-ally committed:
Merge pull request #22 from ryan-ally/snyk-fix-b813e808dc87f65eb16b86beab6dc9c3

[Snyk] Fix for 1 vulnerabilities

ryan-ally committed:
Merge pull request #26 from ryan-ally/snyk-upgrade-c6726f0c5c96f45444df773b7b6ace24

[Snyk] Upgrade source-map from 0.6.1 to 0.7.3

ryan-ally committed:
Merge pull request #25 from ryan-ally/snyk-upgrade-5630f4ffbc6ca03667c8080ae08ffb51

[Snyk] Upgrade: angular, angular-route

ryan-ally committed:
Merge pull request #24 from ryan-ally/snyk-upgrade-4ef2286dd41ed6dbe8a1c8619f454b19

[Snyk] Upgrade zone.js from 0.10.3 to 0.11.3

snyk-bot committed:
fix: upgrade typescript from 4.1.2 to 4.1.3

Snyk has created this PR to upgrade typescript from 4.1.2 to 4.1.3.

See this package in npm:
https://www.npmjs.com/package/typescript

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/69ab45e4-872c-4af1-94ac-186ac0589505?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade @webcomponents/custom-elements from 1.2.1 to 1.4.3

Snyk has created this PR to upgrade @webcomponents/custom-elements from 1.2.1 to 1.4.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/b1517d30-032c-43e2-a7c3-88ade71bdbdd?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade rxjs from 6.5.4 to 6.6.3

Snyk has created this PR to upgrade rxjs from 6.5.4 to 6.6.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/b1517d30-032c-43e2-a7c3-88ade71bdbdd?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - @angular/animations from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/common from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/compiler from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/core from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/elements from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/forms from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/platform-browser from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/platform-browser-dynamic from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/router from 11.0.0 to 11.0.4.
    See this package in npm: 
  - @angular/service-worker from 11.0.0 to 11.0.4.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/b1517d30-032c-43e2-a7c3-88ade71bdbdd?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade tslib from 2.0.0 to 2.0.3

Snyk has created this PR to upgrade tslib from 2.0.0 to 2.0.3.

See this package in npm:


See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/b1517d30-032c-43e2-a7c3-88ade71bdbdd?utm_source=github&utm_medium=upgrade-pr

snyk-bot committed:
fix: upgrade multiple dependencies with Snyk

Snyk has created this PR to upgrade:
  - @angular/cdk from 11.0.0 to 11.0.2.
    See this package in npm: 
  - @angular/material from 11.0.0 to 11.0.2.
    See this package in npm: 

See this project in Snyk:
https://app.snyk.io/org/ryan-ally/project/b1517d30-032c-43e2-a7c3-88ade71bdbdd?utm_source=github&utm_medium=upgrade-pr

ryan-ally committed:
Merge pull request #32 from ryan-ally/snyk-upgrade-0779dc588abbfd7894843e2e6558132b

[Snyk] Upgrade: @angular/cdk, @angular/material

ryan-ally committed:
Merge pull request #31 from ryan-ally/snyk-upgrade-02da7e8d3391c53ef86565bf727245c7

[Snyk] Upgrade tslib from 2.0.0 to 2.0.3

ryan-ally committed:
Merge branch 'master' into snyk-upgrade-302a1e1b3fd122fc9850da7902d4e64b

ryan-ally committed:
Merge pull request #30 from ryan-ally/snyk-upgrade-302a1e1b3fd122fc9850da7902d4e64b

[Snyk] Upgrade: @angular/animations, @angular/common, @angular/compiler, @angular/core, @angular/elements, @angular/forms, @angular/platform-browser, @angular/platform-browser-dynamic, @angular/router, @angular/service-worker

ryan-ally committed:
Merge branch 'master' into snyk-upgrade-c87e2032b3b6c6c04d14f2d7fe5a8853

ryan-ally committed:
Merge pull request #29 from ryan-ally/snyk-upgrade-c87e2032b3b6c6c04d14f2d7fe5a8853

[Snyk] Upgrade rxjs from 6.5.4 to 6.6.3

ryan-ally committed:
Merge branch 'master' into snyk-upgrade-efcbd174ec8a3892f783c17e14ca65a2

ryan-ally committed:
Merge pull request #28 from ryan-ally/snyk-upgrade-efcbd174ec8a3892f783c17e14ca65a2

[Snyk] Upgrade @webcomponents/custom-elements from 1.2.1 to 1.4.3

ryan-ally committed:
Merge pull request #27 from ryan-ally/snyk-upgrade-91f56485ee15f1c1162a061ee702538c

[Snyk] Upgrade typescript from 4.1.2 to 4.1.3

petebacondarwin commented:
Closing as this PR is just spamming the repository with commits, while the author has no CLA signature.
If this is a valid fix that you wish to be merged @ryan-ally - please create a new PR and follow our submission guidelines.

ryan-ally reviewed (commented):
Yup
